storybook/storybook-react/src/components/Heading/Heading.stories.tsx

```diff
@@ -4,55 +4,57 @@
  */
 
 import { Heading } from '@amsterdam/design-system-react/src'
-import { ComponentMeta, ComponentStory } from '@storybook/react'
+import { Meta } from '@storybook/react'
 
 import '@amsterdam/design-system-css/src/heading/heading.scss'
 
 export default {
   title: 'Heading',
   id: 'heading',
   component: Heading,
-} as ComponentMeta<typeof Heading>
-
-const Template: ComponentStory<typeof Heading> = (args) => <Heading {...args} />
-
-export const Heading1 = Template.bind({})
-
-Heading1.args = {
-  children: 'Jouw typograaf biedt mij zulke exquise schreven!',
+  args: {
+    children: 'Jouw typograaf biedt mij zulke exquise schreven!',
+    level: 1,
+    styleLevel: 1,
+  },
+  tags: ['autodocs'],
+} satisfies Meta<typeof Heading>
+
+export const Heading1 = {
+  name: 'Heading 1',
```

Comment:
Can we use an empty object here? We never use this name, right?

Reply:
I believe it's how you set the title of the story

Reply:
Changed it, also tested it, `name` does nothing so I removed it from alle stories. Perhaps it was inherited from CSF2.